packages/table/table.scss

```diff
@@ -36,7 +36,7 @@ $row-end-padding: $component-spacing--large;
     }
 
     &__row {
-        box-shadow: 0 $bottom-border-size 0 0 rgba(0, 0, 0, 0.2);
+        box-shadow: inset 0 -#{$bottom-border-size} 0 0 rgba(0, 0, 0, 0.2);
```

Comment:
Funker det ikke med bare `-$bottom-border-size`? Spiller ingen rolle egentlig, bare lurer på om det også er en quirk 😅

Reply:
Uten interpolasjon? Skulle trodd, men da ble første 0 omitted i Chrome og hele saken havnet ute på høyre side i stedet for under 🤔 Så virker som om den ikke taklet det så godt, uten at jeg klarte å sette noen god pekepinn på hvorfor.

Reply:
Interessant 😅 Best å ha det med da!